crd-diff-config.yaml

```diff
@@ -0,0 +1,109 @@
+checks:
+  crd:
+    scope:
+      enabled: true
+    existingFieldRemoval:
+      enabled: true
+    storedVersionRemoval:
+      enabled: true
+  version:
+    sameVersion:
+      enabled: true
+      unhandledFailureMode: "Closed"
```

Comment:
I think the crd-diff repo needs a docs dir. In that it needs at a minimum a doc outlining how to use the config options. I gleaned from the code for instance that we could doc this field like:

```unhandledFailureMode: Specifies handling for unexpected validation failures. "Closed" indicates strict enforcement, where failures are considered critical.```

I'd be happy to contribute a doc like this over on the tool side of this if it's desired?

Reply:
You're more than welcome to create a PR! I haven't gotten around to writing proper docs yet

Reply:
Added much more detailed documentation for the tool and configuration options. Docs site at https://everettraven.github.io/crd-diff/